plugins/slack/lib/samson_slack/samson_plugin.rb

```diff
@@ -0,0 +1,65 @@
+require 'slack'
+
+module SamsonSlack
+  class Engine < Rails::Engine
+  end
+end
+
+Samson::Hooks.callback :stage_defined do
```

Comment:
this hook is in flux at the moment, sorry about that: https://github.com/zendesk/samson/pull/324/files#diff-54c06d25726a01bbf1b82656eb8655b9R1